05_local-governance/descriptive-representation/descriptive_representation_denom.qmd

```diff
@@ -259,62 +304,79 @@ places_pop <- places_pop %>%
   mutate(state = sprintf("%0.2d", as.numeric(state)))
 places_pop <- places_pop %>%
   mutate(place = sprintf("%0.5d", as.numeric(place)))
+```
+
 
-# create a concatenated GEOID based on state + county & state + place
+Create a concatenated GEOID based on state + county & state + place.
+
+```{r}
 county_pop$GEOID <- paste(county_pop$state,county_pop$county, sep = "")
 
 places_pop$GEOID <- paste(places_pop$state,places_pop$place, sep = "")
+```
 
+Keep the most recent year of population data (not 2022, but 2021).
 
-
-# keep the most recent year of population data (not 2022, but 2020)
+```{r}
 county_pop <- filter(county_pop, year > 2019)
```

Comment:
There's data in the county/place file for 2020, 2021, and 2022 so you are actually keeping 3 years of data, and I think the final dataset is 3X as long as it should be.